java/tools/BUILD

```diff
@@ -0,0 +1,35 @@
+load(
```

Comment:
Can you put all of this in //java/arcs/tools instead of //java/tools? It should really be in the correct namespace.

Reply:
Oops, I had meant to put it there! Done.